Documentation/joss/notes.md

```diff
@@ -17,7 +17,7 @@ Yes, GPLv3.0 - see [licence](../../Licence.txt).
 
 > * The software should have an obvious research application.
 
-See list of publications made with the software [here](./paper.bib).
+See list of publications made with OIP (TUM Open Infrastructure Platform) [here](./paper.bib).
```

Comment:
I thought that the software is called TUM OpenInfraPlatform?

Reply:
lol - fixed